src/components/Results/Results.test.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Results } from '../index';
+
+const testMockItems = [
```

Comment:
you can avoid this decalaration importing the json as you did on the homePage, and appsing one element of the object (or maybe all of them)